packages/oidc/src/refresh/refreshGrant.ts

```diff
@@ -27,8 +27,8 @@ import {
   KeyPair,
   TokenEndpointResponse,
 } from "@inrupt/solid-client-authn-core";
+
 // NB: once this is rebased on #1560, change dependency to core package.
```

Comment:
@NSeydoux you left this comment, but I'm not sure what this was about?

Reply:
I'm not sure either to be honest, this probably can be removed